Libs/Widgets/ctkConsole.cpp

```diff
@@ -339,14 +412,42 @@ void ctkConsolePrivate::keyPressEvent(QKeyEvent* e)
       this->setTextCursor(text_cursor);
       e->accept();
       }
+    this->updateCompleterIfVisible();
+    return;
+    }
+
+  // End of line should be the end of interactive area
```

Comment:
@MayeulChassagnard  This block of code should be in its own commit - could be the same as the one adding `isMoveRighttWithinLine`